src/RIGID/fix_rigid.cpp

```diff
@@ -126,15 +126,36 @@ FixRigid::FixRigid(LAMMPS *lmp, int narg, char **arg) :
   // nbody = # of non-zero ncount values
   // use nall as incremented ptr to set body[] values for each atom
 
-  } else if (strcmp(arg[3],"molecule") == 0) {
+  } else if (strcmp(arg[3],"molecule") == 0 || strcmp(arg[3],"custom") == 0) {
     rstyle = MOLECULE;
-    iarg = 4;
-    if (atom->molecule_flag == 0)
-      error->all(FLERR,"Fix rigid molecule requires atom attribute molecule");
-
+    tagint *molecule;
```

Comment:
this declaration overrides the global declaration as class member in `fix_rigid.h`. is that intentional?

Reply:
Yes, it is intentional. In fact, I should have deleted the declaration as a class member, because this variable is not used anywhere else, neither in `fix_rigid.cpp` nor in any `fix_rigid_n*.cpp`.